### PR TITLE
Added fillter for project owners and mentors 

### LINKED ISSUE #33: Improve the fetchUsers hook!
The fetch Users hook present inside the `/lib` folder needs a small update.

The purpose of the hook is to fetch the list of all the contributors for the project! 
We need to filter out the accounts for our admins `anushbhatia` and `smaranjitghose` and possible mentors including me `abhu-A-J`.

You need to improve the map mehod where we return out all contributors except these 3.

In addition that that crate two new files on the folder `data` from where we can an array of objects describing admin and mentor details (same format which we get from GITHUB API).

### PR DESCRIPTION
**Pull request by Souvikns**
closes #33

### CODE REVIEW

**Comment by abhu-A-J** [comment]
@Souvikns Read the issue details properly you still need to do more make two files where there will be a static list of admins and mentors fro the project.

Also, try doing the task of filtering without using util library, it's an overkill! Ask me on slack if there is confusion! Remove lodash as dependency!

File: src/lib/contributers.js [hunk 1]
```diff
@@ -26,6 +27,10 @@ const useContributerData = props => {
                 })
             })
 
+            contribtorList = filter(contribtorList, el => {
+                return el.username !== 'anushbhatia' && el.username !== 'smaranjitghose' && el.username !== 'abhu-A-J'
+            })
+
```
Comment (abhu-A-J) [inline src/lib/contributers.js:33]: Filter it right here by replacing map with filter. 
Use .includes methods for filtering condition. 
MDN : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Also make a folder in src with name data and add two JS files each exploring a static array of objects for admins and mentors . Fill in these details manually with same information you get from API

Comment (Souvikns) [reply src/lib/contributers.js:33]: I had installed the loadash then realized that I could do it without it. I am reverting the code and filter in map function only

Comment (abhu-A-J) [reply src/lib/contributers.js:33]: Do the second part in here as well for the issue for getting a list of admins and mentors!

**Comment by abhu-A-J** [comment]
@Souvikns do the second part of the issue! Export a static array list for admins and mentors details in a new folder called `data`  inside `src` and have two files for them (adminList.js) and (mentorList.js)

**Comment by Souvikns** [comment]
@abhu-A-J check once, is this ok

**Comment by anushbhatia** [comment]
What is the use for this?

**Comment by Souvikns** [comment]
A page was being created for all the contributors, mentors, and owners. #19 . 
Here the data was static that means every time a new contributor arrives this page had to be updated. So with this hook, we will be getting data from the Github API and so when new contributors arrive, their detail will be updated automatically.

**Comment by abhu-A-J** [comment]
> What is the use for this?

If you hit the API for contributors list we will get all the contributors which includes admins, mentors etc. We find to find only the contributors list and want it to be dynamic since it will keep changing.

The mentors and admins list is pretty static (just few mentors and admins) so we'll add them by ourselves

**Pull request merged**